src/parser/rogue/outlaw/modules/Abilities.js

```diff
@@ -33,15 +33,15 @@ class Abilities extends CoreAbilities {
         gcd: {
           static: standardGcd,
         },
-        enabled: !combatant.hasTalent(SPELLS.SLICE_AND_DICE_TALENT.id),
+        // enabled: !combatant.hasTalent(SPELLS.SLICE_AND_DICE_TALENT.id),
       },
       {
-        spell: SPELLS.SLICE_AND_DICE_TALENT,
+        spell: SPELLS.SLICE_AND_DICE,
         category: Abilities.SPELL_CATEGORIES.ROTATIONAL,
         gcd: {
           static: standardGcd,
         },
-        enabled: combatant.hasTalent(SPELLS.SLICE_AND_DICE_TALENT.id),
+        enabled: combatant.hasTalent(SPELLS.SLICE_AND_DICE.id),
```

Comment:
Is Slice and Dice a talent still? If so then it should be called SLICE_AND_DICE_TALENT. If it is not a talent, then you dont need the enabled option